modules/pel/peltool/config.py

```diff
@@ -7,3 +7,4 @@ def __init__(self) -> None:
         self.allow_plugins = True
         self.serviceable_only = False
         self.non_serviceable_only = False
+        self.critSysTerm = False
```

Comment:
can we mention the variable name as CriticalSysTermination or CriticalSysTermErr ? So, it is easier to understandable ?

Reply:
Tried using same names as was done in [cpp](https://github.com/openbmc/phosphor-logging/blob/master/extensions/openpower-pels/tools/peltool.cpp#L294) implementation.